file-formats/ront/ront-v1.json

```diff
@@ -0,0 +1,130 @@
+{
+  "$comment": "This file is autogenerated, do not edit manually, see https://github.com/SAP/abap-file-formats for more information.",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/ront/ront-v1.json",
+  "title": "General Information",
+  "type": "object",
+  "properties": {
+    "formatVersion": {
+      "title": "ABAP File Format Version",
+      "description": "The ABAP file format version",
+      "type": "string",
+      "const": "1"
+    },
+    "header": {
+      "title": "Header",
+      "description": "Header",
+      "type": "object",
+      "properties": {
+        "description": {
+          "title": "Description",
+          "description": "Description of the ABAP object",
+          "type": "string",
+          "maxLength": 60
+        },
+        "originalLanguage": {
+          "title": "Original Language",
+          "description": "Original language of the ABAP object",
+          "type": "string",
+          "minLength": 2,
+          "maxLength": 2,
+          "pattern": "^[a-z]+$"
+        },
+        "abapLanguageVersion": {
+          "title": "ABAP Language Version",
+          "description": "ABAP language version",
+          "type": "string",
+          "enum": [
+            "standard",
+            "keyUser",
```

Comment:
Does RONT support ABAP language version `keyUser`?

Reply:
I asked Martin and my other colleagues from RAP Services about this. For now RONT does not support ABAP Language Version "keyUser".